src/components/SearchBox.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import { Wrapper } from "./GlobalStyles";
+
+import PropTypes from "prop-types";
+
+export const SearchBox = props => {
+  const [query, setQuery] = useState("");
+  return (
+    <main>
+      <form
+        onSubmit={e => {
+          e.preventDefault();
+          props.onSubmit(query);
+          setQuery("");
+        }}
+      >
+        <label>Start by searching for a phone number</label>
+        <input
+          type="text"
+          placeholder="Type a phone number"
+          onChange={e => setQuery(e.target.value)}
+          value={query}
+        />
```

Comment:
It's not accessible. Either the `input` should be inside the `label` tag or you need to add an `id` on `input` and an `htmlFor` on the `label`.